explorer/client/src/components/pagination/PaginationWrapper.tsx

```diff
@@ -2,19 +2,26 @@
 // SPDX-License-Identifier: Apache-2.0
 import { useState, useCallback } from 'react';
 
+import TabFooter from '../../components/tabs/TabFooter';
+
 import styles from './PaginationWrapper.module.css';
 
 export default function PaginationWrapper({
     results,
     viewComponentFn,
+    ITEMS_PER_PAGE,
```

Comment:
```suggestion
    itemsPerPage,
```

nit: This is not a constant

Reply:
Done